buildspec_sync.yml

```diff
@@ -24,8 +24,8 @@ phases:
       - export AWS_SESSION_TOKEN=`echo $CREDS | jq -r .Credentials.SessionToken`
 
       # Verify from the verification account
-      - './scripts/publish.sh cicd-verify ${AWS_REGION}'
-      - './scripts/publish.sh cicd-verify-ssm ${AWS_REGION}'
+      - './scripts/publish.sh cicd-verify ${AWS_REGION} stable'
+      - './scripts/publish.sh cicd-verify-ssm ${AWS_REGION} stable'
 artifacts:
```

Comment:
What does adding this new argument accomplish? The code does not seem to use it as far as I can tell. I don't think its needed? Since the sync tasks always check stable version now.

Reply:
because we need to know how to set the third parameter here: https://github.com/zhonghui12/aws-for-fluent-bit/blob/pr-stable-tagging/scripts/publish.sh#L611. The same in method `cicd-verify-ssm`